content/en/news/releases/1.8.x/announcing-1.8/upgrade-notes/index.md

```diff
@@ -0,0 +1,120 @@
+as the protocol used by `MySQL`, there will be no initial bytes to sniff. To mitigate this issue in the past, Istio introduced
+a detection timeout. However, we found this caused frequent telemetry and traffic failures during slow connections, while increasing latency
+for misconfigured server first protocols rather than failing fast.
+
+This timeout has been disabled by default. This has the following impacts:
+
+- Non "server first" protocols will no longer have a risk of telemetry or traffic failures during slow connections
+- Properly configured "server first" protocols will no longer have an extra 5 seconds latency on each connection
+- Improperly configured "server first" protocols will experience connection timeouts. Please ensure you follow the steps listed in [Server First Protocols](/docs/ops/configuration/traffic-management/protocol-selection/#server-first-protocols)
+  to ensure you do not run into traffic issues.
+
+## Update AuthorizationPolicy resources to use `remoteIpBlocks`/`notRemoteIpBlocks` instead of `ipBlocks`/`notIpBlocks` if using the Proxy Protocol
+
+If using the Proxy Protocol on a load balancer in front an ingress gateway in conjunction with `ipBlocks`/`notIpBlocks` on an AuthorizationPolicy to perform IP-based access control, then please update the AuthorizationPolicy to use `remoteIpBlocks`/`notRemoteIpBlocks` instead after upgrading. The `ipBlocks`/`notIpBlocks` fields now strictly refer to the source IP address of the packet that arrives at the sidecar.
+
+## `AUTO_PASSTHROUGH` Gateway mode
+
+Previously, gateways were configured with multiple Envoy `cluster` configurations for each Service in the cluster, even those
+not referenced by any `Gateway` or `VirtualService`. This was added to support the `AUTO_PASSTHROUGH` mode on Gateway, generally used for exposing Services across networks.
+
+However, this came at an increased CPU and memory cost in the gateway and Istiod. As a result, we have disabled these by default
+on the `istio-ingressgateway` and `istio-egressgateway`.
+
+If you are relying on this feature for multi-network support, please ensure you apply one of the following changes:
+
+1. Follow our new [Multicluster Installation](/docs/setup/install/multicluster/) documentation.
+
+   This documentation will guide you through running a dedicate gateway deployment for this type of traffic (generally referred to as the `eastwest-gateway`).
+   This `eastwest-gateway` will automatically be configured to support `AUTO_PASSTHROUGH`.
+
+1. Modify your installation of the gateway deployment to include this configuration. This is controlled by the `ISTIO_META_ROUTER_MODE` environment variable. Setting this to `sni-dnat` enables these clusters, while `standard` (the new default) disables them.
+
+{{< text yaml >}}
+ingressGateways:
+- name: istio-ingressgateway
+    enabled: true
+    k8s:
+    env:
+        - name: ISTIO_META_ROUTER_MODE
+        value: "sni-dnat"
```

Comment:
It seems the `value` field is indented incorrectly.

Reply:
fixed